net/mdns-repeater/src/opnsense/mvc/app/views/OPNsense/MDNSRepeater/index.volt

```diff
@@ -31,29 +31,104 @@ POSSIBILITY OF SUCH DAMAGE.
 $( document ).ready(function() {
     var data_get_map = {'general': '/api/mdnsrepeater/settings/get'};
     mapDataToFormUI(data_get_map).done(function(data){
-        formatTokenizersUI();
-        $('select').selectpicker('refresh');
+        //~ formatTokenizersUI();
+        //~ $('select').selectpicker('refresh');
```

Comment:
zap if not needed, or uncomment / finish. at least put a comment if it cannot be finished for the first merge